examples/ewatercycle_api_notebook.ipynb

```diff
@@ -0,0 +1,496 @@
+{
```

Comment:
<p>Nice! the simple mapping of catchment="Meuse" will need something more explicit, I think. Where exactly a catchment ends is not always straightforward, so different models and modellers disagree.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eWaterCycle/ewatercycle/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>